### PR TITLE
Bug 69

### PR DESCRIPTION
**Pull request by MarekBleschke**

### CODE REVIEW

**Comment by magul** [comment]
@MarekBleschke as You can see build in Travis hasn't pass. That shouldn't happen.

**Comment by MarekBleschke** [comment]
@magul problem fixed :)

File: bower.json [hunk 1]
```diff
@@ -1,9 +1,5 @@
 {
   "name": "plantingjs",
   "private": true,
-  "dependencies": {
-    "jquery": "~2.1.1",
-    "jquery-ui": "~1.11.4",
-    "backbone": "~1.2.1"
-  }
+  "dependencies": {}
 }
```
Comment (magul) [inline bower.json:5]: it looks, that we don't use bower at all. Could this file be removed?

**Comment by magul** [comment]
@cmwd there is review waiting for You. :)

**Comment by cmwd** [comment]
:+1:

**Pull request merged**